modules/3_helpernode/helpernode.tf

```diff
@@ -38,6 +38,8 @@ locals {
         bastion_master_ip   = var.bastion_ip[0]
         bastion_backup_ip   = length(var.bastion_ip) > 1 ? slice(var.bastion_ip, 1, length(var.bastion_ip)) : []
         forwarders      = var.dns_forwarders
+        lb_ipaddr       = var.lb_ipaddr
+        ext_dns         = var.ext_dns
```

Comment:
This is intentional? I guess helpernode node does not support these vars yet.

Reply:
These are added for support external DNS/LB. Remove from them and will create a new commit for them.